library/src/scala/annotation/publicInBinary.scala

```diff
@@ -0,0 +1,14 @@
+package scala.annotation
+
+/** A binary API is a definition that is annotated with `@publicInBinary` or overrides a definition annotated with `@publicInBinary`.
+ *  This annotation can be placed on `def`, `val`, `lazy val`, `var`, `object`, and `given` definitions.
```

Comment:
The SIP also says that class constructors are supported, but this isn't mentioned here.

Reply:
Added doc